cs

```diff
@@ -27,7 +27,11 @@ var camera = new MjpegCamera({
   url: program.url,
   name: typeof program.name === 'function' ? '' : program.name
 });
-camera.start();
+
+console.log("Starting...");
```

Comment:
Please remove the logging or genericize it.

Reply:
Oops, just some left over test code. Removed :)